devolutions-gateway/src/rdp_proxy.rs

```diff
@@ -4,7 +4,7 @@ use std::time::Duration;
 
 use crate::api::kdc_proxy::send_krb_message;
 use crate::config::Conf;
-use crate::config::dto::{DomainUser, KerberosServer};
+use crate::config::dto::KerberosConfig as KrbConfig;
```

Comment:
suggestion: As a rule of thumb, avoid renaming imports using `as`. It hurts readability.